docs/Maths.md

```diff
@@ -47,6 +47,76 @@ Result:
 
 ![](matplotlib.png)
 
+### Using Pandas ###
+Create a plot using [pandas.DataFrame.plot](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.plot.html)
+
+```python
+from fpdf import FPDF
+import pandas as pd
+import matplotlib.pyplot as plt
+import io
+
+data = {'Unemployment_Rate': [6.1,5.8,5.7,5.7,5.8,5.6,5.5,5.3,5.2,5.2],
+        'Stock_Index_Price': [1500,1520,1525,1523,1515,1540,1545,1560,1555,1565]
+       }
+
+plt.figure() # Store figure
```

Comment:
What do you mean by "Store" here?

Reply:
I think it little bit ambiguous, so I change the comment with 
`# Create a new figure object`

Reply:
Perfect!